### PR TITLE
Collada export bones

### PR DESCRIPTION
**Pull request by wise86-android**
add the ability to export joints/bone in the collada files

### CODE REVIEW

File: code/ColladaExporter.cpp [hunk 1]
```diff
@@ -1136,21 +1136,23 @@ void ColladaExporter::WriteJointsWeightSourceNode(const size_t pIndex){
 	delete [] tempBoneWeight;
 }
 
+template <typename T>
+static void moveMatrixInArray(const aiMatrix4x4t<T> &matrix, T *outBuffer){
+	std::memcpy(outBuffer,matrix[0],16*sizeof(T));
```
Comment (acgessler) [inline code/ColladaExporter.cpp]: should this not be &matrix[0]?

Also wondering if perhaps a copy with two loops is better to maintain.

**Comment by acgessler** [comment]
Apologies for late response. If you're still there: definitely interested in merging this. Can you rebase?

File: code/ColladaExporter.cpp (hunk 1 above)
Comment (wise86-android) [inline code/ColladaExporter.cpp]: No matrix[0] return the pointer to the data buffer for be able to use matrix[i][j].

see https://github.com/assimp/assimp/blob/master/include/assimp/matrix4x4.inl#L241

**Comment by kimkulling** [comment]
I merged the latest master into this one and created a new pull-request. When the new pull-request was merged I will close this on. Thanks a lot for the help.

**Pull request closed without merging**